### PR TITLE
Updated links for migrated repositories:

### PR DESCRIPTION
**Pull request by dannys42**
- Kitura-Sample
 - Kitura-Compression
 - Kitura-WebSocket-NIO
 - Kitura-WebSocket-Compression
 - Kitura-CredentialsJWT
 - Kitura-CredentialsFacebook

### CODE REVIEW

**Review by ddunn2 — APPROVED** [review]

File: content/docs/guides/authentication/jwt.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ title: JSON Web Token Authentication
...
+[Swift-JWT](https://github.com/IBM-Swift/Swift-JWT) is our implementation of JSON Web Token using Swift. It allows you to create, sign and verify JWTs on iOS, macOS and Linux using a range of algorithms.  [Kitura-CredentialsJWT](https://github.com/Kitura/Kitura-CredentialsJWT) is a JWT plugin to use with the existing Kitura-Credentials package that offers both Codable and Raw routing methods for easily authenticating JWTs. This guide will demonstrate how to use Swift-JWT and Kitura-CredentialsJWT to implement Single Sign On (SSO) authentication for your Kitura routes. This will allow a user to sign in once and then to access resources from other routes without having to repeat the authentication process.
```
Comment (dannys42) [inline content/docs/guides/authentication/jwt.md:9]: Link update here?

File: content/docs/guides/websockets/echo-server.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ touch main.swift
 
 ## Step 2: Setting up the Server
 
-Add the following content to your `Package.swift` file, replacing "x.x.x" with the latest releases of [Kitura](https://github.com/IBM-Swift/Kitura/releases), [HeliumLogger](https://github.com/IBM-Swift/HeliumLogger/releases), [Kitura-WebSocket](https://github.com/IBM-Swift/Kitura-WebSocket/releases) and [Kitura-WebSocket-NIO](https://github.com/IBM-Swift/Kitura-WebSocket-NIO/releases):
+Add the following content to your `Package.swift` file, replacing "x.x.x" with the latest releases of [Kitura](https://github.com/IBM-Swift/Kitura/releases), [HeliumLogger](https://github.com/IBM-Swift/HeliumLogger/releases), [Kitura-WebSocket](https://github.com/IBM-Swift/Kitura-WebSocket/releases) and [Kitura-WebSocket-NIO](https://github.com/Kitura/Kitura-WebSocket-NIO/releases):
```
Comment (dannys42) [inline content/docs/guides/websockets/echo-server.md:33]: More link updates?

**Review by dannys42 — COMMENTED** [review]
Looks like a few links are still pointing to IBM-Swift?

**Comment by futurejones** [comment]
@dannys42 I don't think we need this PR anymore as these changes were included in #448

**Pull request closed without merging**